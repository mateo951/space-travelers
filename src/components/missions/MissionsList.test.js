import { Provider } from 'react-redux';
import { render, screen } from '@testing-library/react';
import store from '../../redux/configureStore';
import MissionList from '../../pages/Missions';

describe('Mission component', () => {
  describe('Use Jest snapshots to test the Missions component', () => {
    it('renders Missions component as expected', async () => {
      const MissionsComponent = render(
        <Provider store={store}>
          <MissionList />
        </Provider>,
      );
      expect(await screen.findAllByText('Description')).toBeTruthy();
      expect(MissionsComponent.asFragment()).toMatchSnapshot();
    });
  });
});
