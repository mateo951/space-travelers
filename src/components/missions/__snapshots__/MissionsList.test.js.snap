// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component Use Jest snapshots to test the Missions component renders Missions component as expected 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="inner-container"
    >
      <div
        class="mainDiv"
      >
        <table
          class="test table table-striped table-bordered"
        >
          <tbody>
            <tr>
              <th>
                Mission
              </th>
              <th>
                Description
              </th>
              <th>
                Status
              </th>
              <th>
                Action
              </th>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
